### ISSUE TITLE
issue about sample

### ISSUE THREAD

**Issue by forthelich** [Red-Folder/bgs-sample#5]
Hi , firstly my English bad as my Java. I don't understand, which method ran in background? dowork() or others? Thx

**Comment by Red-Folder** [comment]
Yes it is the doWork().  Any background work should be within this method
